syz-ci/manager.go

```diff
@@ -440,11 +440,11 @@ func (mgr *Manager) reportBuildError(rep *report.Report, info *BuildInfo, imageD
 	req := &dashapi.BuildErrorReq{
 		Build: *build,
 		Crash: dashapi.Crash{
-			Title:       rep.Title,
-			Corrupted:   false, // Otherwise they get merged with other corrupted reports.
-			Maintainers: rep.Maintainers,
-			Log:         rep.Output,
-			Report:      rep.Report,
+			Title:      rep.Title,
+			Corrupted:  false, // Otherwise they get merged with other corrupted reports.
+			Recipients: rep.Recipients,
```

Comment:
If clients send the info in the new way, then we need to update dashboard to understand this new way first (in this this change or ahead of this change). Or otherwise we should send the info to the dashboard in the old way.
If we deploy this, things will break.